device/create.go

```diff
@@ -0,0 +1,69 @@
+package device
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/jinzhu/copier"
+	"github.com/traggo/server/auth"
+	"github.com/traggo/server/auth/rand"
+	"github.com/traggo/server/generated/gqlmodel"
+	"github.com/traggo/server/model"
+	"github.com/traggo/server/user/password"
+)
+
+var (
+	timeNow          = time.Now
+	randToken        = rand.Token
+	comparePassword  = password.ComparePassword
+	errUserPassWrong = errors.New("username/password combination does not exist")
```

Comment:
As a user I always hate to read such error messages, but I guess this is also for security reasons, so an attacker doesn't know if he is on the right track...